Fix /plugin command

Czompi opened the pull request:
Remove the unnecessary comma (from the end of the list) from the plugin list and added subcommand '/pl'.

Czompi committed:
Fix /plugin command

Remove the unnecessary comma (from the end of the list) from the plugin list and added subcommand '/pl'.

Craftplacer reviewed (approved):
Could be refactored, but seems fine to me.

Czompi committed:
Modified /plugin command (rev.2)

Migrated the plugin ChatMessage to variable and modified the comma ChatMessage as well.

Czompi committed:
Added maxPlayers to config

Czompi committed:
Merge branch 'master' into master

Seb-stian reviewed (approved):
I like it 👍
